metrics_gather.sh

```diff
@@ -331,32 +331,39 @@ PIXELS=$(($WIDTH*$HEIGHT*$FRAMES))
 
 echo "$x" "$PIXELS" "$SIZE"
 
-PSNR=$(cat "$BASENAME-psnr.out" | grep Total)
+#PSNR=$(cat "$BASENAME-psnr.out" | grep Total)
```

Comment:
We would require to commend out L327
`"$DUMP_PSNR" -a "$FILE" "$BASENAME.y4m" > "$BASENAME-psnr.out"` 
to make sure it is not computing PSNR